packages/cache/RELEASES.md

```diff
@@ -46,4 +46,4 @@
 
 ### 1.0.9
   - Use @azure/ms-rest-js v2.6.0
-  - Use @azure/storage-blob v12.8.0
+  - Use @azure/storage-blob v12.8.0
```

Comment:
What's with this `cache` change showing up in `tool-cache` 🤔

Reply:
Hmm maybe that's like a newline diff or something. The text is the same.